10_nwis_pull/src/nwis_pull.R

```diff
@@ -0,0 +1,109 @@
+#### pull ####
+
+# prepare a plan for downloading (from NWIS) and posting (to GD) one data file
+# per state
+plan_nwis_pull <- function(partitions, service) {
+
+  folders <- list(
+    tmp='10_nwis_pull/tmp',
+    out='10_nwis_pull/out',
+    log='10_nwis_pull/log')
```

Comment:
in addition, only `tmp` is used in this script. Maybe you can delete the others?

Reply:
I don't think the tmp folder needs to be exposed since the goal is to not have to interact with these individual files. 

I modified to get rid of the other files, though.